op-node/rollup/derive/calldata_source.go

```diff
@@ -173,10 +172,10 @@ func DataFromEVMTransactionsV2(config *rollup.Config, txs types.Transactions, re
 			receipt := receipts[j]
 			data := tx.Data()
 			// Exclude transactions if L1 transaction did call submit function.
-			if len(data) < 4 || !bytes.Equal(data[:4], SubmitAbi.ID) {
-				log.Warn("tx sent to inbox contract did not call submit function", "index", j)
-				continue // not calling submit function, ignore
-			}
+			//if len(data) < 4 || !bytes.Equal(data[:4], SubmitAbi.ID) {
```

Comment:
The comment above should say `Exclude transactions if L1 transaction did not call submit function`. Currently, the "not" is missing.

Reply:
Noted, fixed in 202514e7c31c91c359008e8120945f0f3c79b5c4.